charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           claimName: {{ template "flink-job-cluster.fullname" . }}-savepoint
   {{- end }}
   {{- if .Values.jobManager.volumes }}
-    {{ toYaml .Values.jobManager.volumes | indent 6}}
+    {{- toYaml .Values.jobManager.volumes | nindent 6}}
```

Comment:
Hard to see in this PR - but you have 4 spaces here in the beginning, and you are indenting by 6.. i think you want those to match. So you either want it to be 4 or 6 depending on the format.

Reply:
Rendered locally here what I got, 
with nindent 6
```
    volumes:
      - hostPath:
          path: /temp/savepoints
          type: DirectoryOrCreate
        name: local-savepoint-dir
    volumeMounts:
      - mountPath: /temp/savepoints
        name: local-savepoint-dir
```
with nindent 4
```
    volumes:
    - hostPath:
        path: /temp/savepoints
        type: DirectoryOrCreate
      name: local-savepoint-dir
    volumeMounts:
    - mountPath: /temp/savepoints
      name: local-savepoint-dir
```
it seems effect is the same, but format is diff

Reply:
looks `nindent 6` is consistent better with other indentation in place, as it should be one level under `volumes` that has 4 spaces

Reply:
The number of spaces you are indenting is specific to the key in the Yaml that you are trying to insert data into. The key you're inserting into is at https://github.com/Nextdoor/k8s-charts/blob/f9f87badf9d099a7c9178cc1b6ec6366f697d665/charts/flink-job/templates/flinkjobcluster.yaml#L43 and has 4 spaces in front of it. Because it's a YAML Array, you can EITHER indent it 4 or 6 spaces, but you cannot mix and match. The "savepoint" blob (https://github.com/Nextdoor/k8s-charts/blob/f9f87badf9d099a7c9178cc1b6ec6366f697d665/charts/flink-job/templates/flinkjobcluster.yaml#L44-L47) is indenting 6 spaces it looks like.

So the current code looks like this:
```yaml
{{- if or (.Values.jobManager.volumeMounts) (.Values.savepoints.enabled) }}
    volumeMounts:
  {{- if .Values.savepoints.enabled }}
      - name: savepoint-storage
        mountPath: {{ .Values.savepoints.savepointDir }}
  {{- end }}
  {{- if .Values.jobManager.volumeMounts }}
    {{- toYaml .Values.jobManager.volumeMounts | nindent 6}}
  {{- end }}
{{- end }}
```

I would reformat it ... the fact that some of the conditions are "less indented" than the text they then create is really confusing. Instead, I'd do this:

```yaml
    {{- if or (.Values.jobManager.volumeMounts) (.Values.savepoints.enabled) }}
    volumeMounts:
      {{- if .Values.savepoints.enabled }}
      - name: savepoint-storage
        mountPath: {{ .Values.savepoints.savepointDir }}
      {{- end }}
      {{- with .Values.jobManager.volumeMounts }}
      {{- toYaml . | nindent 6}}
      {{- end }}
    {{- end }}
```

By indenting everything to the proper level, it's easier to understand what the indent levels are supposed to look like.